pkg/scaffold/resource/resource.go

```diff
@@ -67,8 +67,14 @@ func (r *Resource) Validate() error {
 	if err := IsDNS1123Subdomain(r.Group); err != nil {
 		return fmt.Errorf("group name is invalid: (%v)", err)
 	}
-	// Check if the version is a valid value
-	versionMatch := regexp.MustCompile("^v\\d+(alpha\\d+|beta\\d+)?$")
+
+	// Remove the . and - from the group to do GO imports in the scaffold project
+	r.GroupImportSafe = strings.Replace(r.Group, "-", "", -1)
```

Comment:
what's up with this change here?

Reply:
I added in the first comment all fixes made. 
Note that was just fixed the basic ones to add the lint.

The following is the issue fixed here. 

> pkg/scaffold/resource/resource.go:71:18: S1007: should use raw string (`...`) with regexp.MustCompile to avoid having to escape twice (gosimple)
> 	versionMatch := regexp.MustCompile("^v\\d+(alpha\\d+|beta\\d+)?$")

Reply:
just use a raw string:

```go
regexp.MustCompile(`^v\d+(alpha\d+|beta\d+)?$`)
```

(raw strings are the kind surrounded by backticks instead of double quotes)

Reply:
True :-) I was very silly now :-P

Reply:
Done.